web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
```

Comment:
How is this TournamentPlayer related to the existing Player model, if at all? I'm envisioning the following: A User will have a Player on the "Real Life Go Server", and that player is the entity that is participating in a tournament. Is this what you guys had in mind?

Maybe the whole thing should just be the bare minimum fields relevant to the player as he relates to the tournament, and one foreign key to the actual Player model?